### PR TITLE
Added Bluzelle chain

### PR DESCRIPTION
**Pull request by njmurarka**
Added bluzelle chain, as per osmosis instructions.

### CODE REVIEW

File: bluzelle/chain.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      "v2.0"
+    ],
+    "cosmos_sdk_version": "0.44.3",
+    "tendermint_version": "0.34.14",
+    "genesis": {
+      "genesis_url": "https://a.client.sentry.net.bluzelle.com:26657/genesis"
+    }
+  },
+  "logo_URIs": {
+    "jpg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.jpg",
```
Comment (JeremyParish69) [inline bluzelle/chain.json:47]: Please remove the jpg. Only png and svg are accepted.

File: bluzelle/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          "denom": "bnt",
+          "exponent": 6
+        }
+      ],
+      "base": "ubnt",
+      "name": "Bluzelle",
+      "display": "blz",
+      "symbol": "BLZ",
+      "logo_URIs": {
+        "jpg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.jpg",
```
Comment (JeremyParish69) [inline bluzelle/assetlist.json:22]: please remove jpg

File: bluzelle/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+          "denom": "elt",
+          "exponent": 6
+        }
+      ],
+      "base": "uelt",
+      "name": "ELT",
+      "display": "elt",
+      "symbol": "ELT",
+      "logo_URIs": {
+        "jpg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/bluzelle.jpg",
```
Comment (JeremyParish69) [inline bluzelle/assetlist.json:49]: please remove jpg, and use a different logo for the different token (I hope they don't use the same logo...)

File: bluzelle/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+          "denom": "g4",
+          "exponent": 6
+        }
+      ],
+      "base": "ug4",
+      "name": "G4",
+      "display": "g4",
+      "symbol": "G4",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluzelle/images/g4.png",
```
Comment (JeremyParish69) [inline bluzelle/assetlist.json:75]: Strongly preferred to have an exact square aspect ratio for the image to avoid morphing in various displays

**Comment by njmurarka** [comment]
On these things. Will be resolved quickly. Thanks!

File: bluzelle/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "bnt",
+          "exponent": 6
+        }
+      ],
+      "base": "ubnt",
+      "name": "Bluzelle",
+      "display": "blz",
```
Comment (JeremyParish69) [inline bluzelle/assetlist.json:19]: the 'display' denom must be listed under denom_units, but 'blz' is not listed there. Is the 'bnt' and 'ubnt' a typo?

Comment (JeremyParish69) [reply bluzelle/assetlist.json:19]: checked out the explorer. Looks like ubnt is correct. In which case I suggest changing the 'display' denom to 'bnt'. But also adding 'blz' as an alias unit of 'bnt'.

Comment (njmurarka) [reply bluzelle/assetlist.json:19]: It is not a typo. BLZ is what we have been using as a symbol, but I understand now. I have corrected it and will push an update shortly along with the other fixes.

File: bluzelle/assetlist.json (hunk 3 above)
Comment (njmurarka) [reply bluzelle/assetlist.json:49]: No image yet for ELT. Can I still list it and have nothing in the logo_URIs array?

File: bluzelle/chain.json [hunk 6]
```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "bluzelle",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://bluzelle.com/",
+  "pretty_name": "Bluzelle",
+  "chain_id": "bluzelle-8",
+  "bech32_prefix": "bluzelle",
```
Comment (JeremyParish69) [inline bluzelle/chain.json:9]: Please register this bech32 prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: bluzelle/assetlist.json (hunk 5 above)
Comment (JeremyParish69) [reply bluzelle/assetlist.json]: yes, with the denom_unit, e.g., 
```
{
  "denom": "bnt",
  "exponent": 6,
  "aliases": [
    "blz"
  ]
}
```

File: bluzelle/chain.json (hunk 6 above)
Comment (njmurarka) [reply bluzelle/chain.json:9]: Actually, it was registered about 12 hrs ago. 

https://github.com/satoshilabs/slips/commit/2457b96b3fde88066de82810ac86aec9370ad1ea

**Pull request merged**